protobuf/model_config.proto

```diff
@@ -1377,6 +1408,16 @@ message ModelSequenceBatching
   //@@     model.
   //@@
   repeated ControlInput control_input = 2;
+
+  //@@  .. cpp:var:: State state (repeated)
+  //@@
+  //@@     The implicit state that should be stored in Triton for performing
+  //@@     inference requests in a sequence. Each sequence holds an implicit
+  //@@     state local to itself. The output state provided by the model in 
```

Comment:
The language appears little confusing saying the link is between "output_name" and "input_name" fields. We must be more explicit saying the tensor contents get transferred and not just the "*_name" fields.

Reply:
I improved the wording. Let me know if it is still unclear.